msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/PrivateMessagesTool.java

```diff
@@ -2731,6 +2760,14 @@ private PrivateMessage processPvtMsgReplyAllSendHelper(boolean preview, Boolean
 		  }
 		  return null ;
 	  }
+	  if(StringUtils.isEmpty(getForwardBody())) {
```

Comment:
does `StringUtils.isBlank()` make more sense?

Reply:
All the similar comparisons in the file are using isEmpty, maybe it's acceptable to send a blank space if you don't want to add anything else... But I can change it.